sal_interfaces/CBP/CBP_Commands.xml

```diff
@@ -115,4 +103,23 @@
             <Count>1</Count>
         </item>
     </SALCommand>
+    <SALCommand>
+        <Subsystem>CBP</Subsystem>
+        <Version>3.8</Version>
+        <Author>ecoughlin</Author>
+        <EFDB_Topic>CBP_command_unpark</EFDB_Topic>
+        <Alias>unpark</Alias>
+        <Device></Device>
+        <Property></Property>
+        <Action></Action>
+        <Value></Value>
+        <Explanation>http://sal.lsst.org</Explanation>
+        <item>
+            <EFDB_Name>value</EFDB_Name>
+            <Description>Attribute required by the SAL API, but is unused.</Description>
```

Comment:
This is no longer needed (and has not been for quite awhile). Please remove this and all other ignored attributes that were only added to make old SAL happy.